P/pandoc/build_tarballs.jl

```diff
@@ -2,7 +2,7 @@ using BinaryBuilder
 
 # Collection of pre-build pandoc binaries
 name = "pandoc"
-pandoc_ver = "2.19.2"
+pandoc_ver = "2.19.3"
 version = VersionNumber(pandoc_ver)
```

Comment:
```suggestion
pandoc_ver = "2.19.2"
version = v"2.9.13" # <-- This version number is a lie to allow using aarch64-apple-darwin
```

Reply:
Caught that when I was making a similar change to `pandoc_citeproc`. Fixed in [5c2209a](https://github.com/JuliaPackaging/Yggdrasil/pull/6220/commits/5c2209a1e04bef7493197e5465b20b0192b2269d).